docs/Troubleshooting.md

```diff
@@ -378,3 +381,27 @@ After downloading the license file perform the following steps:
 Information about licenses from the [SAS Viya Platform Operations Guide](https://documentation.sas.com/?cdcId=itopscdc&cdcVersion=default&docsetId=k8sag&docsetTarget=n14rkqa3cycmd0n1ub50k47x7lbb.htm)
 
 Note, these steps are only applicable for updating your license file, if you are going to be updating the SAS deployment or including additional products in your order we recommend that your perform your update manually. See this note in the [README](https://github.com/sassoftware/viya4-deployment#updating-sas-viya-manually)
+
+
+## PostgreSQL Data Transfer Guide - Post Migration Steps
+
+### Symptom:
+
+You had an existing SAS Viya platform deployment that was created using the viya4-deployment project and configured to use Crunchy as the database, and you have now completed the manual steps outlined in the "PostgreSQL Data Transfer Guide" to move your data over to an external PostgreSQL cluster. You now want to use the viya4-deployment project to manage your installation again.
```

Comment:
Are we providing links to this doc - "PostgreSQL Data Transfer Guide" - or just mentioning it and then have the customer look for the doc.

Reply:
I was actually going to add a note about that, but forgot to. 
The documentation is not yet available externally on go.documentation.sas.com so I do not have a link I could here yet. I am thinking about creating another small ticket to update the link when it does go live, assuming the 19th.

Reply:
Links added.